infrastructure/helm-chart/charts/apps/charts/api/templates/security.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: security
+  namespace: {{ .Values.global.namespace }}
+data:
+  jwtSecret: {{ .Values.jwtSecret | default (randAlphaNum 128) | quote }}
+  systemToken: {{ .Values.systemToken | default (randAlphaNum 24) | quote }}
+  allowedOrigins: {{ .Values.allowedOrigins | quote }}
```

Comment:
Once we want to have these values optional on `airy create` time, we can just remove them from here and create an empty configMap.